faroe/api/src/main/java/org/n52/faroe/dao/InMemoryServiceDaoImpl.java

```diff
@@ -0,0 +1,26 @@
+package org.n52.faroe.dao;
```

Comment:
License header is missing

Reply:
isn't it auto-generated? any idea why some of the files have been missed out?